app/src/main/java/com/tehmou/rxbookapp/data/stores/StoreModule.java

```diff
@@ -0,0 +1,37 @@
+package com.tehmou.rxbookapp.data.stores;
+
+import android.content.ContentResolver;
+
+import javax.inject.Singleton;
+
+import dagger.Module;
+import dagger.Provides;
+
+/**
+ * Created by Pawel Polanski on 5/16/15.
+ */
+@Module
+public final class StoreModule {
+
+    @Provides
+    @Singleton
```

Comment:
I mean the other stores are not singletons, should they be as well? The UserStore is indeed a global store that is not contextualized to any actual user.